subsystems/indexersubsystem.py

```diff
@@ -11,51 +13,144 @@ class Mode(Enum):
-            self.stagingMotor,
-            constants.kForwardSensorStaging,
-            constants.kSimStagingSensorId,
-        )
+        print(f"Initializing Falcon: {constants.kStagingMotorName}")
+        if not ctreCheckError(
+            "configFactoryDefault",
+            self.stagingMotor.configFactoryDefault(
+                constants.kConfigurationTimeoutLimit
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kP",
+            self.stagingMotor.config_kP(
+                constants.kStagingMotorPIDSlot,
+                constants.kStagingMotorPGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kI",
+            self.stagingMotor.config_kI(
+                constants.kStagingMotorPIDSlot,
+                constants.kStagingMotorIGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kD",
+            self.stagingMotor.config_kD(
+                constants.kStagingMotorPIDSlot,
+                constants.kStagingMotorDGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "configForwardLimitSwitchSource",
+            self.stagingMotor.configForwardLimitSwitchSource(
+                LimitSwitchSource.Deactivated,
+                LimitSwitchNormal.Disabled,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "configReverseLimitSwitchSource",
+            self.stagingMotor.configReverseLimitSwitchSource(
+                LimitSwitchSource.Deactivated,
+                LimitSwitchNormal.Disabled,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        print(f"{constants.kStagingMotorName} Falcon Initialization Complete")
+
+        self.indexerSensor = self.stagingMotor.isRevLimitSwitchClosed
+        self.stagingSensor = self.stagingMotor.isFwdLimitSwitchClosed
         self.state = self.Mode.Holding
 
     def periodic(self) -> None:
         if self.state == self.Mode.FeedForward:
-            self.indexerMotor.setSpeed(constants.kIndexerSpeed)
-            self.stagingMotor.setSpeed(constants.kStagingSpeed)
+            self.indexerMotor.set(
+                ControlMode.Velocity,
+                constants.kIndexerSpeed * constants.kTalonVelocityPerRPM,
+            )
+            self.stagingMotor.set(
+                ControlMode.Velocity,
+                constants.kStagingSpeed * constants.kTalonVelocityPerRPM,
+            )
         elif self.state == self.Mode.Holding:
-            self.indexerMotor.setSpeed(constants.kIndexerSpeed)
-            self.stagingMotor.setSpeed(-constants.kStagingSpeed)
```

Comment:
since we're using the sensors again shouldn't we re-add the relevant code to make them work?

Reply:
To your earlier point since they haven't been tested as working on the real bot I held off on using them in the state machine. That's not really the point of this PR anyway. Once the sensors are working we may want to use them even more in our sequencing than we have in the past, which should come in a focused PR.